### PR TITLE
Heroku deployable and graph updates

### PR DESCRIPTION
**Pull request by ramhound**
Website deployable on heroku. Pie graph compares data from the panels. Graph bug fixes.

### CODE REVIEW

**Comment by iangudger** [comment]
You should add an [app.json](https://devcenter.heroku.com/articles/app-json-schema) so that the Postgres database is automatically deployed with the app. Example [here](https://github.com/comforme/comforme/blob/master/app.json).

You should also add a [deploy to Heroku button](https://devcenter.heroku.com/articles/heroku-button) to the README.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "website",
+  "version": "1.0.0",
+  "description": "This is a preliminary setup for our seads Frontend website, which will eventually include d3 visualizations. Currently, this website will only work with a local postgres database, which can be installed with the instructions below.",
```
Comment (iangudger) [inline package.json]: Where are the instructions?

See my other comment--no need to create the database manually.

File: settings.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
+
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ['SECRET_KEY']
```
Comment (iangudger) [inline settings.py:26]: Make your  [app.json](https://devcenter.heroku.com/articles/app-json-schema) auto generate a SECRET_KEY when deploying.

File: settings.py [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'seadssystems@gmail.com'
+EMAIL_HOST_PASSWORD = os.environ['EMAIL_HOST_PASSWORD']
+EMAIL_PORT = 587
+
+
+# for password reset
+ADMINS =(('Seads', 'seadssystems@gmail.com'),)
```
Comment (iangudger) [inline settings.py]: Better not to put emails in the code. Maybe make this an environmental variable as well?

File: settings.py [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+# https://warehouse.python.org/project/whitenoise/
+
+STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
+
+
+LOGIN_REDIRECT_URL = '/'
+
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'seadssystems@gmail.com'
```
Comment (iangudger) [inline settings.py]: Better not to put emails in the code. Maybe make this an environmental variable as well?

**Comment by ramhound** [comment]
Will make postgres database deploy automatically soon (after finals)

**Comment by ramhound** [comment]
Postgres database now created upon deployment. Added a deploy to heroku button.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+
+To deploy with Heroku simply click this button, log in/create a free Heroku account and click the "deploy for free" button at the bottom of the page.
+
 #Django Frontend
 This is a preliminary setup for our seads Frontend website, which will eventually include d3 visualizations. Currently, this website will only work with a local postgres database, which can be installed with the instructions below.
```
Comment (iangudger) [inline README.md:6]: Does this need to be updated? Which database is it referring to?

File: package.json (hunk 1 above)
Comment (aadabi) [inline package.json]: @ramhound and @ianlofs , how can we fix this?

**Pull request merged**